Jenkinsfile

```
@@ -24,6 +24,8 @@ pipeline {
     SNAPSHOT = 'true'
     TERRAFORM_VERSION = "0.13.7"
     XPACK_MODULE_PATTERN = '^x-pack\\/[a-z0-9]+beat\\/module\\/([^\\/]+)\\/.*'
+    KIND_VERSION = 'v0.11.1'
+    K8S_VERSION = 'v1.18.2'
```

How about trying to be inline with https://github.com/elastic/beats/blob/8.0/deploy/kubernetes/Jenkinsfile.yml#L21 and use v1.21.1 instead (latest version supported)?

---

I used the one that was default in the CI workers. so let' me update it:

```suggestion
    K8S_VERSION = 'v1.21.1'
```